autospec/pkg_integrity.py

```diff
@@ -754,6 +759,8 @@ def get_integrity_file(package_path):
 
 
 def check(url, download_path, interactive=True):
+    global PUBKEY_PATH
+    PUBKEY_PATH = os.path.join(download_path, "{}.pkey")
```

Comment:
This feels not right. I'd think that the GPGVerifier should probably store the PUBKEY_PATH in the object instead of having it set in check and be global (and also having the string contain an embedded format specifier).

from_disk and from_url are the only callers who need to create the object and both have the path in their argument list. Also reduces a global, what do you think?

Reply:
Agree, done